relay-server/src/metrics_extraction/event.rs

```diff
@@ -48,11 +48,11 @@ impl Extractable for Span {
 pub fn extract_metrics(
     event: &Event,
     spans_extracted: bool,
-    config: &MetricExtractionConfig,
+    config: CombinedMetricExtractionConfig,
```

Comment:
Baby nit: I like these "generic" lifetimes to be explicit.

```suggestion
    config: CombinedMetricExtractionConfig<'_>,
```